_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,200 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the [document title](https://www.w3.org/TR/html5/single-page.html#dom-tree-accessors) in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `<html>` element **and** the `<title>` element contains [text nodes](https://www.w3.org/TR/dom/#text) that do not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
```

Comment:
Why don't specify "the first" directly in the Applicability , as in the #440 ? 

>  This rule applies to the **first** [document title](https://www.w3.org/TR/html5/single-page.html#dom-tree-accessors) ...

Reply:
I think it's because we are using the definition of "document title", and there is only one of them in a document per the specification...
Otherwise, I think we would have to say instead that the rule applies to the first `title` element - but then it wouldn't be clear why it's only the first, unless we use the document title definition to explain. So i think it would be the same amount of explanation in the end...